core/src/test/scala/integration/kafka/api/TransactionsTest.scala

```diff
@@ -763,7 +763,10 @@ class TransactionsTest extends IntegrationTestHarness {
   }
 
   @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
```

Comment:
TestWithParameterizedQuorumAndGroupProtocolNames does not support disabling TV2, I wonder if we want to remove the "true" in the csv list.

Reply:
I'm also wondering if we need both the name and the csv list.